### ISSUE TITLE
Removing 'Fork me on Github' ribbon for mobile devices

### ISSUE THREAD

**Issue by waridrox** [publiclab/community-toolbox#444]
### Please describe the problem (or idea)
For mobile screens that have such small screen real estate to begin with, the **Fork me on Github** ribbon takes almost 30% of the screen size and continues till the end of the page. 

Before the fix
<img width="301" alt="Screenshot 2021-03-22 at 10 52 03 PM" src="https://user-images.githubusercontent.com/58583793/112031639-92728780-8b61-11eb-8ed3-e6e2c0c04e66.png">

After the fix
<img width="320" alt="Screenshot 2021-03-22 at 10 55 20 PM" src="https://user-images.githubusercontent.com/58583793/112031755-b03fec80-8b61-11eb-862e-95daf4352747.png">

If the ribbon is removed, this would utilise the wasted space and make the font bigger that would be easier on the eyes :D

### Please show us where to look

https://code.publiclab.org/

**Comment by govindgoel** [comment]
@waridrox That is there for a reason to easily navigate to the project.

**Comment by waridrox** [comment]
Agreed, but in mobile view, there's already a github logo down at the bottom which directs to the codebase. And someone navigating on mobile device would rarely open the github repo to see the codebase, so I think it would make sense to hide it for small screen resolutions...🤔

<img width="312" alt="Screenshot 2021-03-27 at 12 40 46 PM" src="https://user-images.githubusercontent.com/58583793/112713134-a9b2cb80-8ef9-11eb-8e14-d001ef0b1a5e.png">

**Comment by govindgoel** [comment]
For only a small screen size, we can think of that.

**Comment by Anshul-target** [comment]
Could you assign this project to me .I am very sure I can  do it

**Comment by tarunashri6** [comment]
@waridrox Hi, I would  like to work on this issue.
